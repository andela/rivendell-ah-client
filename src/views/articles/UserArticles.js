const UserArticles = () =>{
    <div>
        <h2>Articles of a particular user</h2>
    </div>
    };
    export default UserArticles;