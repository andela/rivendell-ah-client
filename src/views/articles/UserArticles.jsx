import React from 'react';
import ArticleView from '../../components/ArticleView';

const UserArticles = () => (
  <ArticleView slug="currentSlug" />
);
export default UserArticles;
