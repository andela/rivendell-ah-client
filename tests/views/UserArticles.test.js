import { shallow } from 'enzyme';
import React from 'react';
import chai from 'chai';
import UserArticles from '../../src/views/articles/UserArticles';

describe('testing NewArticle Copmonent', () => {
  describe('basic components it renders', () => {
    it('should render an EditArticleControls component ', () => {
      const mountedObj = shallow(<UserArticles />);
      expect(mountedObj)
        .toMatchSnapshot();
    });
  });
});
