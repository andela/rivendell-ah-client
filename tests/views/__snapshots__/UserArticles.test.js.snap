// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing NewArticle Copmonent basic components it renders should render an EditArticleControls component  1`] = `
<div>
  <h2>
    Articles of a particular user
  </h2>
</div>
`;
